### PR TITLE
Updating v1.1.2 benchmarks to match 2.0 changes (set matching baseline)

### PR DESCRIPTION
**Pull request by jorive**
This is an test/infrastructure only change to enable running previous release 1.1 with the same benchmarks on master and release 2.0.
- Update from the old xUnit Performance runner to the xUnit Performance Api
- Update JIT benchmarks

### CODE REVIEW

File: dependencies.props [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <!-- Full package version strings that are used in other parts of the build. -->
   <PropertyGroup>
     <CoreClrPackageVersion>1.1.3-servicing-25416-01</CoreClrPackageVersion>
-    <XunitPackageVersion>2.1.0</XunitPackageVersion>
+    <XunitPackageVersion>2.2.0-beta2-build3300</XunitPackageVersion>
```
Comment (DrewScoggins) [inline dependencies.props]: Have you tried running the functional tests with this change?  I could see this being a problem.

File: perf.groovy [hunk 2]
```diff
@@ -4,151 +4,308 @@ import jobs.generation.*;
...
+def projectFolder = projectName + '/' + Utilities.getFolderName(branch)
 
 def static getOSGroup(def os) {
-    def osGroupMap = ['Ubuntu':'Linux',
+    def osGroupMap = ['Ubuntu14.04':'Linux',
         'RHEL7.2': 'Linux',
         'Ubuntu16.04': 'Linux',
         'Debian8.4':'Linux',
-        'Fedora23':'Linux',
+        'Fedora24':'Linux',
```
Comment (DrewScoggins) [inline perf.groovy]: I would probably just add Fedora24 and Ubuntu here instead of replacing it.  Seems safer then replacing something.

**Comment by DrewScoggins** [comment]
@dotnet-bot help

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT x64 perf
@dotnet-bot test Windows_NT x86 perf
@dotnet-bot test Ubuntu14.04 perf

**Comment by jorive** [comment]
@dotnet-bot test ci please
@dotnet-bot test Windows_NT x64 perf
@dotnet-bot test Windows_NT x86 perf
@dotnet-bot test Ubuntu14.04 perf

File: tests/scripts/perf-prep.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+# we don't match we delete the line.  This returns just the address of the last nupkg to curl.
+curl "http://benchviewtestfeed.azurewebsites.net/nuget/FindPackagesById()?id='Microsoft.BenchView.JSONFormat'" | grep "content type" | sed "$ s/.*src=\"\([^\"]*\)\".*/\1/;tx;d;:x" | xargs curl -o benchview.zip
+unzip -q -o benchview.zip -d ./tests/scripts/Microsoft.BenchView.JSONFormat
+
+# Install python 3.5.2 to run machinedata.py for machine data collection
+python3.5 --version
+python3.5 ./tests/scripts/Microsoft.BenchView.JSONFormat/tools/machinedata.py
+
+if [ $throughput -eq 1 ]; then
+    # Clone corefx
```
Comment (adiaaida) [inline tests/scripts/perf-prep.sh]: I've just updated this, could you take the changes I made?

**Comment by jorive** [comment]
@dotnet-bot test ci please
@dotnet-bot test Windows_NT x64 perf
@dotnet-bot test Windows_NT x86 perf
@dotnet-bot test Ubuntu14.04 perf

**Comment by jorive** [comment]
@dotnet-bot test ci please
@dotnet-bot test Windows_NT x64 perf
@dotnet-bot test Windows_NT x86 perf
@dotnet-bot test Ubuntu14.04 perf

**Comment by jorive** [comment]
@dotnet-bot
test ci please
test Windows_NT x64 perf
test Windows_NT x86 perf
test Ubuntu14.04 perf

**Comment by jorive** [comment]
@dotnet-bot
test ci please
test Windows_NT x64 perf
test Windows_NT x86 perf
test Ubuntu14.04 perf

**Comment by jorive** [comment]
@dotnet-bot
test ci please
test Windows_NT x64 perf
test Windows_NT x86 perf
test Ubuntu14.04 perf

**Comment by jorive** [comment]
@dotnet-bot
test ci please
test Windows_NT x64 perf
test Windows_NT x86 perf
test Ubuntu14.04 perf

**Comment by adiaaida** [comment]
@dotnet-bot test Linux ARM Emulator Cross Debug Build please

**Comment by jorive** [comment]
@dotnet-bot
test ci please
test Windows_NT x64 perf
test Windows_NT x86 perf
test Ubuntu14.04 perf

**Pull request merged**